src/sql/azdata.proposed.d.ts

```diff
@@ -1741,9 +1741,181 @@ declare module 'azdata' {
 		ownerUri: string;
 	}
 
+	export interface SchemaCompareParams {
+		sourceEndpointInfo: SchemaCompareEndpointInfo;
+		targetEndpointInfo: SchemaCompareEndpointInfo;
+		taskExecutionMode: TaskExecutionMode;
+		deploymentOptions: DeploymentOptions;
+	}
+
+	export interface SchemaCompareGenerateScriptParams {
```

Comment:
this also a duplicate definition removed in the initial schema compare PR

Reply:
same. resolved